doc/installing.rst

```diff
@@ -1,222 +1,316 @@
-is via `pip <https://pypi.org/project/pip/>`_. It is very highly recommended to first set up a clean environment
-in which to install ``PypeIt`` so that possible dependency conflicts can be avoided. This can be done via ``virtualenv``:
+To install the latest release of ``PypeIt`` and its required dependencies, execute
+either
 
-        virtualenv pypeit
-        source pypeit/bin/activate
+.. code-block:: console
 
-or via ``conda``:
+    pip install "pypeit[pyqt5]"
 
-        conda create -n pypeit python=3.8
-        conda activate pypeit
+to select the ``PyQT5`` QT bindings or
 
-See the `Virtualenv documentation <https://virtualenv.pypa.io/en/latest/>`_ and/or `Managing Environments with Conda
-<https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`_ for more details. See also
-`virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_ as an option for more easily managing
-``virtualenv`` virtual environments.
+.. code-block:: console
 
-To install the latest release of ``PypeIt`` and its required dependencies, do either:
+    pip install "pypeit[pyside2]"
 
-        pip install "pypeit[pyqt5]"
+to select ``PySide2``; see :ref:`interactive`.
 
-to select the ``PyQT5`` QT bindings or:
+If you are using KWCI, you will also need the `shapely`_ package. It can be installed by
+including it in the optional dependencies, e.g.:
 
-        pip install "pypeit[pyside2]"
+.. code-block:: console
 
-to select ``PySide2``.
+    pip install "pypeit[pyside2,shapely]"
 
-If you are using KWCI, you may also need the ``shapely`` package. It can be installed by
-including it in the optional dependencies, e.g.:
+.. note::
 
-        pip install "pypeit[pyside2,shapely]"
+    Whether or not it is correct syntax to use the quotes in the commands above
+    depends on your shell.  The above commands are specific to ZShell, whereas
+    you don't need the quotes in Bash.  But, in any case, you should avoid
+    copying these commands from your browser since the unicode for quotation
+    marks may not be correct, leading to errors when they are directly pasted
+    into a terminal window.
 
-Installing via ``conda``
-------------------------
+Install via ``conda``
+---------------------
 
-`Conda <https://docs.conda.io/projects/conda/en/latest/index.html>`_ is
-a popular and widely-used package and environment manager. We provide a yaml file that can be
-used to setup a conda environment called ``pypeit`` that contains all of the required dependencies.  To use this:
+`conda`_ is a popular and widely-used package and environment manager. We
+provide a yaml file that can be used to setup a conda environment called
+``pypeit`` that contains all of the required dependencies.  To use this:
 
- #. Download `environment.yml <https://github.com/pypeit/PypeIt/blob/master/environment.yml>`__.
+    #. Download `environment.yml
+       <https://github.com/pypeit/PypeIt/blob/master/environment.yml>`__.
```

Comment:
You may want to change "master" to "release", although currently github is nice enough to redirect

Reply:
Good catch!  Changed.